strict-equality.js

```diff
@@ -0,0 +1,5 @@
+const strictEqual = (a, b) => {
+  return typeof a == typeof b && a == b
+}
```

Comment:
Very nice! One note on additional clean up you could consider. Since this is a one line return function, you can use fat arrow's implicit return.

```JavaScript
const strictEqual = (a, b) => typeof a == typeof b && a == b
```